.github/workflows/macos.yaml

```diff
@@ -12,8 +16,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - platform: 'osx-64'
-          - platform: 'osx-arm64'
+          - platform: 'x86_64'
```

Comment:
Was this just for readability or for something else as well?

Reply:
For consistency to get the package by its name at the end. If you prefer, switching back to *.pkg would avoid this renaming

Reply:
thanks, I adjusted it slightly to be more explicit